ChainSafe/web3.unity
# Ipfs upload duplicate fix

Made an error throw if the web request returns an error status due to duplicate files already existing, displaying the message to the user also. closes #951 

Webgl still needs to be done as it requires a JS function to go into the compiler conditional but i'll fix that after the marketplace. Added a workaround to fix builds for now.

To test:
Upload a file twice using the GUI in the sample scene.

Expected result:
Error is thrown on duplicate upload.
